flutter/example/lib/main.dart

```diff
@@ -14,16 +15,30 @@ Future<void> main() async {
   await SentryFlutter.init(
     (options) {
       options.dsn = _exampleDsn;
-      // use breadcrumb tracking of WidgetsBindingObserver
```

Comment:
This is not needed anymore.

Reply:
could be removed in a different PR as this is not going to be merged anyway

Reply:
could be removed in a different PR, this PR is not going to be merged anyway